python/pipeline/utils/h5.py

```diff
@@ -94,6 +94,106 @@ def read_behavior_file(filename):
 
     return data
 
+def read_patchmaster(F):
+    if type(F)==str:
+        F=F[:-4]+'%d.h5' # chanage oneCell10_01.h5 to oneCell10_%d.h5
+        # Append '%d.h5' to filename in place of trailing 'x.h5'
```

Comment:
usually the comments describing what you are doing go before the line:
```python3
# This describes the following block of code
skslsl
slslslls

# this describes the next block of code
slslskjfsj
sksslsl 
slslslslsl   # specific comment for this tricky line
```
so it this case it will be 
```python
if ....
    # Append '%d.h5' to filename in place of trailing 'x.h5'
    F = F[:-4] + '%d.h5'
else: 
    skslls
````